examples/platform/silabs/SiWx917/args.gni

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+declare_args() {
+  enable_heap_monitoring = false
+
+  # OTA timeout in seconds
+  ota_periodic_query_timeout_sec = 86400
+
+  # Wifi related stuff - they are overridden by gn -args="use_wf200=true"
```

Comment:
Could this comment be expanded? Arguments themselves would not be overriden, however I am guessing these go unused for wf200 ?

 Also probably not a fault of this PR, so you can resolve and fix in some other PR if needed.